<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="YASUs-MacBook-Pro.local" ignored="7" name="Default test" tests="1" failures="0" timestamp="27 Apr 2021 04:01:32 GMT" time="2.334" errors="0">
  <testcase name="verifyTwoFactorAuthWithInvalidCredentials" time="2.297" classname="AlertAware.api.test.SignIn"/>
  <testcase name="logoutCurrentUser" time="0.0" classname="AlertAware.api.test.SignIn">
    <ignored/>
  </testcase> <!-- logoutCurrentUser -->
  <testcase name="loginAsAUserWithValidCredentials" time="0.0" classname="AlertAware.api.test.SignIn">
    <ignored/>
  </testcase> <!-- loginAsAUserWithValidCredentials -->
  <testcase name="get2fa" time="0.0" classname="AlertAware.api.test.SignIn">
    <ignored/>
  </testcase> <!-- get2fa -->
  <testcase name="verifyTwoFactorAuth" time="0.0" classname="AlertAware.api.test.SignIn">
    <ignored/>
  </testcase> <!-- verifyTwoFactorAuth -->
  <testcase name="logoutInvalidUser" time="0.0" classname="AlertAware.api.test.SignIn">
    <ignored/>
  </testcase> <!-- logoutInvalidUser -->
  <testcase name="loginAsAUserWith2fa" time="0.0" classname="AlertAware.api.test.SignIn">
    <ignored/>
  </testcase> <!-- loginAsAUserWith2fa -->
  <testcase name="loginAsAUserWithInValidCredentials" time="0.0" classname="AlertAware.api.test.SignIn">
    <ignored/>
  </testcase> <!-- loginAsAUserWithInValidCredentials -->
</testsuite> <!-- Default test -->
